cmd/submit.go

```diff
@@ -86,10 +88,18 @@ func submitSolution(
 	*leetcode.SubmitCheckResult,
 	error,
 ) {
+	modifiers, err := getPostModifiers(gen)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get post modifiers: %w", err)
+	}
+
 	solution, err := lang.GetSolutionCode(q)
```

Comment:
Can we integrate the pre-submit modification directly into the `lang.GetSolutionCode` function? What do you think?